SPECS/hiredis/hiredis.spec

```diff
@@ -0,0 +1,192 @@
+Summary:        Minimalistic C client library for Redis
+Name:           hiredis
+Version:        1.0.2
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/redis/hiredis
+Source0:        https://github.com/redis/hiredis/archive/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  openssl-devel
+BuildRequires:  redis
+
+%description
+Hiredis is a minimalistic C client library for the Redis database.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    devel
+This package contains libraries and header files for
+developing applications that use %{name}.
+
+%prep
+%autosetup -p1
+
+%build
+%make_build PREFIX="%{_prefix}" LIBRARY_PATH="%{_lib}"     \
+            DEBUG="%{optflags}" LDFLAGS="%{?__global_ldflags}" USE_SSL=1
+
+%install
+%make_install PREFIX="%{_prefix}" LIBRARY_PATH="%{_lib}"
+
+find %{buildroot} -name '*.a' -delete -print
+
+%check
+make check
```

Comment:
For some reason this seems to be breaking.
Did you validate the test locally ?
PS: Add the option `RUN_CHECK=y` to your `make build-packages ...` command for local validation.

Reference:[link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=407757&view=logs&j=28fbb862-d6a2-53ee-8718-d863c03f3031&t=917769c8-295b-5cb7-fb22-a789faaea754&l=1501)
![image](https://github.com/microsoft/CBL-Mariner/assets/4929861/85dcfdff-2e75-4224-8887-d3c826e75459)

Reply:
Hi @mfrw,
This is a known issue in 1.0.2 https://github.com/redis/hiredis/issues/1088. Since the check part was getting ignored in the build and the build is successful, can we ignore this.

I can add a comment regarding this in the spec file.